Gopkg.toml

```diff
@@ -118,3 +118,11 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/GaryBoone/GoStats"
```

Comment:
Do we need this dependency? From what I understand, we replaced the library for statistics.